rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -90,7 +91,7 @@ bool emu_settings::Init()
 	return true;
 }
 
-void emu_settings::LoadSettings(const std::string& title_id)
+void emu_settings::LoadSettings(const std::string& title_id, const std::string& game_path)
```

Comment:
the game_path seems irrelevant as you already pass the self name as seríal to get_custom_config_path in other places